addons/edi/models/edi_synchronizer.py

```diff
@@ -370,7 +381,10 @@ def execute(self):
                     vals_list = [rec.target_values(rec._record_values()) for rec in batch]
                     if doc.fail_fast:
                         for rec, vals in zip(batch, vals_list):
-                            rec[target].write(vals)
+                            if rec._changed_vals:
```

Comment:
Will it be any record without _changed_vals? I think if there isnt any change we dont yield in in elide, so practically we yield only the ones with changes isnt it?